CHANGES.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 
+- cohttp-eio: Make server response type abstract and allow streaming in cohttp-eio (talex5 #1024)
```

Comment:
```suggestion
- cohttp-eio: allow streaming in cohttp-eio (talex5 #1024)
- cohttp: make server response type abstract (talex5 #1024)
```
This is also a change in cohttp signature itself, I think we should make it clear

Reply:
Good point - I've updated it as suggested.